lib/commons/table/get-headers.js

```diff
@@ -57,7 +57,7 @@ function traverseForHeaders(headerType, position, tableGrid) {
  * @return {Array<HTMLTableCellElement>} Array of headers associated to the table cell
  */
 table.getHeaders = function(cell, tableGrid) {
-	if (cell.hasAttribute('headers')) {
+	if (cell.getAttribute('headers')) {
 		return commons.dom.idrefs(cell, 'headers');
```

Comment:
That doesn't fix the whole problem. The issue lays in this line right here, not the one above. There are two problems with this line:

1. It finds things that aren't cells in the table
2. if it doesn't find anything, it shouldn't return empty here, but continue down to look for row/ column headers.

Reply:
1 is already covered by td-headers-attr rule so we don't need to fail it in this rule, thus we can just assume if it has an attribute (we can trim) then it can pass this rule. td-headers-attr (part 2 change) then can validate the headers attr is correct.

Reply:
I don't follow... td-headers-attr fails if the headers attribute is incorrect. We shouldn't also fail td-has-headers if there are fallback headers, like this:

```html
<tr> <th>Hello</th> <td headers="non-existant">goodbye</td> </tr>
```

I think this should pass `td-has-headers` and it should fail `td-headers-attr`.